twilight-validate/src/component/component_v2.rs

```diff
@@ -53,20 +61,74 @@ pub const THUMBNAIL_DESCRIPTION_LENGTH_MAX: usize = 1024;
 pub fn component_v2(component: &Component) -> Result<(), ComponentValidationError> {
     match component {
         Component::ActionRow(ar) => action_row(ar, true)?,
+        Component::Label(l) => label(l)?,
         Component::Button(b) => button(b)?,
         Component::Container(c) => container(c)?,
         Component::MediaGallery(mg) => media_gallery(mg)?,
         Component::Section(s) => section(s)?,
         Component::SelectMenu(sm) => select_menu(sm)?,
         Component::TextDisplay(td) => text_display(td)?,
-        Component::TextInput(ti) => text_input(ti)?,
+        Component::TextInput(ti) => text_input(ti, true)?,
```

Comment:
Should this be allowed on root level? I think text input is only allowed in modals, where it should not be at root level.
Should I change this in this PR?

Reply:
Sure.